specification/cognitiveservices/AnomalyDetector/tspconfig.yaml

```diff
@@ -17,7 +17,7 @@ options:
     examples-directory: ./examples
     omit-unreachable-types: true
     output-file: openapi.json
-    emitter-output-dir: "{project-root}/../data-plane/AnomalyDetector/stable"
+    emitter-output-dir: "{project-root}/../data-plane/AnomalyDetector/stable/v1.1"
```

Comment:
@markcowl: Should `output-file` and `emitter-output-dir` be changed to align with the sample?  Something like this:

```
    emitter-output-dir: "{project-root}/.."
    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
```

In particular, I thought `output-file` is supposed to use a template expression after the recent breaking change:

https://azure.github.io/typespec-azure/docs/release-notes/release-2023-04-11#azure-toolstypespec-autorest-config-interpolation

Reply:
Also, why is the version `v1.1` instead of a date-based version like `2022-11-01-preview`?